pkg/operator/target_config_reconciler.go

```diff
@@ -55,6 +62,21 @@ func NewTargetConfigReconciler(
 	kubeClient kubernetes.Interface,
 	eventRecorder events.Recorder,
 ) *TargetConfigReconciler {
+	// make sure our list of excluded system namespaces is up to date
+	allNamespaces, err := kubeClient.CoreV1().Namespaces().List(ctx, metav1.ListOptions{})
+	if err != nil {
+		klog.ErrorS(err, "error listing namespaces")
+		return nil
+	}
+	for _, ns := range allNamespaces.Items {
+		if matched, err := regexp.Match(`^openshift-[\w\d-]*`, []byte(ns.Name)); matched {
+			excludedNamespaces = append(excludedNamespaces, ns.Name)
```

Comment:
Can you make `excludedNamespaces` a part of `TargetConfigReconciler` to avoid accidental writes to it? I am not fan of global variables that are not read-only after their initialization.

Reply:
done